python-package/lightgbm/plotting.py

```diff
@@ -646,6 +655,22 @@ def create_tree_digraph(
     example_case : numpy 2-D array, pandas DataFrame or None, optional (default=None)
         Single row with the same structure as the training data.
         If not None, the plot will highlight the path that sample takes through the tree.
+    max_category_values : int, optional (default=None)
+        The maximum number of category values to display in tree nodes.
```

Comment:
We should probably also state here that if the number of thresholds is greater than this value it will be collapsed and shown in the tooltip instead.

Reply:
done, please have a look at the modification